SPEC/FILES.md

```diff
@@ -1,141 +1,41 @@
 # Files API
 
-> The files API enables users to use the File System abstraction of IPFS.
-
+> The files API enables users to use the File System abstraction of IPFS. There are two Files API, one at the top level, the original `add`, `cat`, `get` and `ls`, and another behind the [`files`, also known as MFS](https://github.com/ipfs/specs/issues/98). [We are currently going through a revamping process of this APIs to make them more user friendly](https://github.com/ipfs/interface-ipfs-core/issues/284).
+
+- [add](#add)
+- [addReadableStream](#addreadablestream)
+- [addPullStream](#addpullstream)
+- [addFromFs](#addFromFs)
+- [addFromUrl](#addFromFs)
+- [cat](#cat)
+- [catReadableStream](#filescatreadablestream)
+- [catPullStream](#filescatpullstream)
+- [get](#filesget)
+- [getReadableStream](#filesgetreadablestream)
+- [getPullStream](#filesgetpullstream)
 - [ls](#ls)
 - [lsReadableStream](#lsreadablestream)
 - [lsPullStream](#lspullstream)
```

Comment:
Moving all the non MFS commands to the top level. This preps the users for the future Files API as they will get used to call the add, cat, get and ls from there, but when the MFS parts get coalesced with the non MFS parts, they will just get the lovely surprise that there will be a "drive" looking thing that contains all the files they added.